static/styles/settings.css

```diff
@@ -78,7 +78,7 @@ h3 .fa-question-circle-o {
         #pw_strength {
             width: 140px;
             height: 8px;
-            margin: 6px 0 0 0;
+            margin: 6px 0 0;
```

Comment:
Please remove extraneous formatting changes from your commit.

(See our guides on [tidying your commit history](https://zulip.readthedocs.io/en/latest/git/using.html#examine-and-tidy-your-commit-history), [force-pushing your changes](https://zulip.readthedocs.io/en/latest/git/using.html#force-push-changes-to-github-after-you-ve-altered-your-history), and [updating a pull request](https://zulip.readthedocs.io/en/latest/git/pull-requests.html#update-a-pull-request); do not delete your branch or open a different pull request to make changes.)

Reply:
Got it!